core/domain/email_manager.py

```diff
@@ -339,9 +454,25 @@ def send_moderator_action_email(
     """Sends a email immediately following a moderator action (publicize,
     unpublish, delete) to the given user.
 
-    The caller is responsible for ensuring that emails are allowed to be sent
-    to users (i.e. feconf.CAN_SEND_EMAILS is True).
+    Raises an exception if emails are not allowed to be sent to users (i.e.
+    feconf.CAN_SEND_EMAILS is False).
+
+    Args:
+        sender_id: str. User ID of the sender.
+        recipient_id: str. User ID of the recipient.
+        intent: str. The intent string (cause/purpose) of the email.
+        exploration_title: str. The title of the exploration
+                            on which the moderator action was taken.
```

Comment:
Indent to 4 chars after the previous line; move some words to the end of the previous line.